pkg/lib/common.js

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
What about `pkg/lib/cockpit` although that is also kinda nasty. But having `common` cockpit things in `pkg/lib` feels a bit strange

Reply:
I agree with the sentiment and moved everything to `pkg/lib/_internal` as a result.